src/main/java/pageUrl/PageUrl.java

```diff
@@ -0,0 +1,5 @@
+package pageUrl;
```

Comment:
Packages names are lowercased. If a name contains 2 words, then it should be no space (pageurl) or words should be separated by dot (page.url).
For this case package is not needed at all. PageUrl class can be directly placed under main/java.